### PR TITLE
Update configuring cluster-wide proxy section for ALBO

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/OCPBUGS-11399

Link to docs preview: https://58231--docspreview.netlify.app/openshift-enterprise/latest/networking/aws_load_balancer_operator/configure-egress-proxy-aws-load-balancer-operator.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by xenolinux** [comment]
@lihongan @alebedev87 Can you please take a look at this PR?

File: modules/configuring-egress-proxy.adoc [hunk 1]
```diff
@@ -24,20 +24,20 @@ $ oc -n aws-load-balancer-operator label cm trusted-ca config.openshift.io/injec
...
 ----
-$ oc -n aws-load-balancer-operator exec deploy/aws-load-balancer-operator-controller-manager -c manager -- ls -l /etc/pki/tls/certs/albo-tls-ca-bundle.crt
+$ oc -n aws-load-balancer-operator exec deploy/aws-load-balancer-operator-controller-manager -c manager -- bash -c "ls -l /etc/pki/tls/certs/albo-tls-ca-bundle.crt; printenv TRUSTED_CA_CONFIGMAP_NAME"
 ----
 +
 .Example output
 [source,terminal]
 ----
--rw-r--r--. 1 root 1000690000 5875 Jan 11 12:25 /etc/pki/tls/certs/albo-tls-ca-bundle.crt
+-rw-r--r--. 1 root 1000690000 5875 Jan 11 12:25 /etc/pki/tls/certs/albo-tls-ca-bundle.crt trusted-ca
```
Comment (alebedev87) [inline modules/configuring-egress-proxy.adoc:40]: `trusted-ca` should be on the new line.

**Review by alebedev87 — COMMENTED** [review]
LGTM, just 1 remark.

**Comment by lihongan** [comment]
LGTM

File: modules/configuring-egress-proxy.adoc [hunk 2]
```diff
@@ -6,7 +6,7 @@
 [id="nw-configuring-cluster-wide-proxy_{context}"]
 = Configuring the AWS Load Balancer Operator to trust the certificate authority of the cluster-wide proxy
 
-. Create the config map to contain the certificate authority (CA) bundle in the `aws-load-balancer-operator` namespace by running the following command:
+. Create the config map to contain the certificate authority (CA) bundle in the `aws-load-balancer-operator` namespace and inject a CA bundle trusted by {product-title} into a configmap by running the following command:
```
Comment (kcarmichael08) [inline modules/configuring-egress-proxy.adoc:9]:
```suggestion
. Create the config map to contain the certificate authority (CA) bundle in the `aws-load-balancer-operator` namespace and inject a CA bundle that is trusted by {product-title} into a configmap by running the following command:
```

**Review by kcarmichael08 — COMMENTED** [review]
One tiny suggestion - lgtm!

File: modules/configuring-egress-proxy.adoc [hunk 3]
```diff
@@ -6,7 +6,7 @@
 [id="nw-configuring-cluster-wide-proxy_{context}"]
 = Configuring the AWS Load Balancer Operator to trust the certificate authority of the cluster-wide proxy
 
-. Create the config map to contain the certificate authority (CA) bundle in the `aws-load-balancer-operator` namespace by running the following command:
+. Create the config map to contain the certificate authority (CA) bundle in the `aws-load-balancer-operator` namespace and inject a CA bundle that is trusted by {product-title} into a configmap by running the following command:
```
Comment (jboxman-rh) [inline modules/configuring-egress-proxy.adoc:9]: @xenolinux, we're calling it _config map_: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#config-map

**Pull request merged**